src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
sorry, not sure what are `modus`

> We could execute the config file, but this won't be able if php-cs-fixer is ran in PHAR modus. 

not true.

> isn't it more logical if it runs in intersect modus without a finder

no, it intersect you need to have at least two collections. here, it's cli args + finder from config.